Cargo.toml

```diff
@@ -52,6 +52,7 @@ with-serde-support = ["serde", "serde_derive", "num/serde"]
 [dev-dependencies]
 criterion = "0.4"
 rand = "0.8.5"
+lazy_static = "1"
```

Comment:
This absolutely isn't a hard requirement for the `approx` tests, but it's useful. Happy to remove it.